app/controllers/provider_interface/reports/reference_status_reports_controller.rb

```diff
@@ -0,0 +1,24 @@
+module ProviderInterface
+  module Reports
+    class ReferenceStatusReportsController < ProviderInterfaceController
+      attr_reader :provider
+
+      def show
+        @provider = current_user.providers.find(provider_id)
+        @provider_application_references = @provider
+          .application_references
+          .joins(application_form: :application_choices)
+          .where.not(application_choices: { accepted_at: nil })
+          .select('"references".name, "references".feedback_status,"application_choices".id as application_choice_id, "application_forms".first_name as first_name, "application_forms".last_name as last_name')
+
+        @provider_application_references = @provider_application_references.page(params[:page]).per(25)
```

Comment:
We need to handle the provider putting in an invalid page parameter. It currently returns an empty report structure but i think it would be better to either always redirect to the last page of the results or return a message like we do with the [applications table component](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/components/support_interface/applications_table_component.html.erb#L6)